onnxruntime/core/optimizer/double_qdq_pairs_remover.cc

```diff
@@ -45,21 +46,16 @@ bool DoubleQDQPairsRemover::IsNodeRemovable(
       self->OpType() != "DequantizeLinear" ||
       self->GetInputEdgesCount() != 1 ||
       self->GetOutputEdgesCount() != 1 ||
-      self->InputDefs().size() != InputIndex::TOTAL_COUNT ||
```

Comment:
Removed this condition because the call to IsQDQPairSupported (below) already checks that the zero-point input is present.